velox/functions/prestosql/window/WindowFunctionsRegistration.cpp

```diff
@@ -18,23 +18,26 @@
 
 namespace facebook::velox::window {
 
+extern void registerAggregateWindowFunction(const std::string& name);
```

Comment:
Why do you need 'extern' here?

Reply:
extern is needed since all those functions are in cpp files. Their definitions are not available via header files.